rasa/cli/arguments/train.py

```diff
@@ -43,29 +53,76 @@ def set_train_core_arguments(parser: argparse.ArgumentParser):
     compare_arguments = parser.add_argument_group("Comparison Arguments")
     add_compare_params(compare_arguments)
 
+    add_arguments_for_training_in_chunks(parser)
+
 
-def set_train_nlu_arguments(parser: argparse.ArgumentParser):
-    add_config_param(parser)
-    add_domain_param(parser, default=None)
-    add_out_param(parser, help_text="Directory where your models should be stored.")
+def set_train_nlu_arguments(parser: argparse.ArgumentParser) -> None:
+    """Sets the command line arguments for the command 'rasa train nlu'.
 
-    add_nlu_data_param(parser, help_text="File or folder containing your NLU data.")
+    Args:
+        parser: the parser to add the arguments to
+    """
+    default_arguments.add_config_param(parser)
+    default_arguments.add_domain_param(parser, default=None)
+    default_arguments.add_out_param(
+        parser, help_text="Directory where your models should be stored."
+    )
+
+    default_arguments.add_nlu_data_param(
+        parser, help_text="File or folder containing your NLU data."
+    )
 
     add_num_threads_param(parser)
 
     add_model_name_param(parser)
     add_persist_nlu_data_param(parser)
 
+    add_arguments_for_training_in_chunks(parser)
+
 
-def add_force_param(parser: Union[argparse.ArgumentParser, argparse._ActionsContainer]):
+def add_arguments_for_training_in_chunks(parser: argparse.ArgumentParser) -> None:
+    """Adds the arguments for training in chunks.
+
+    Args:
+        parser: the parser to add the arguments to
+    """
+    train_in_chunks_arguments = parser.add_argument_group(
+        "Arguments for training in chunks"
+    )
+
+    train_in_chunks_arguments.add_argument(
+        "--in-chunks",
+        action="store_true",
+        help="Trains your Rasa model in smaller chunks.",
+    )
+    train_in_chunks_arguments.add_argument(
+        "--number-of-chunks", type=int, default=5, help="Number of chunks to use.",
+    )
```

Comment:
these arguments seem a bit redundant. How about just using `--number-of-chunks` with a default of `1` and tell them to increase this number when they run in memory issues?

Reply:
We might want to add more command line arguments later, so I used a flag. But we can also just go with `--number-of-chunks` and return to the flag once it is needed.